zuul.d/projects.yaml

```diff
@@ -144,3 +144,14 @@
       - ansible-ee-tests
       - publish-to-galaxy
       - publish-to-automation-hub
+
+- project:
+    name: github.com/ansible-collections/kubernetes.core
+    default-branch: main
+    merge-mode: squash-merge
+    templates:
+      - system-required
+      - ansible-collections-kubernetes-core-units
+      - network-ee-tests
```

Comment:
Can you use `ansible-ee-tests` instead of `network-ee-tests`.

`ansible-ee-tests` is a refactored version of `network-ee-tests` that can work on SF.